src/tests/integration/server.itest.js

```diff
@@ -513,6 +513,27 @@ describe('/api/monsters', () => {
       expect(res.body.results[0].name).toEqual(name);
     });
   });
+  describe('with challenge rating query', () => {
+    it('returns objects with only one provided challenge rating', async () => {
```

Comment:
**[Nitpick]:** This might just be holdover from writing rspec, but I might structure this like so:

```javascript
describe('with only one provided challenge rating', () => {
  it('returns expected objects', async () => {
```

Reply:
Fixed